pkg/api/authz.go

```diff
@@ -133,9 +143,37 @@ func (ac *AccessController) getContext(username string, request *http.Request) c
 	return ctx
 }
 
+func getUserGroup(ac *AccessController, username string) string {
```

Comment:
I think there's also the case where is user is part of multiple groups, and the permissions cumulate.